clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesMutatingWebhookConfigurationHandler.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.op.handler;
+
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.caching.agent.KubernetesCoreCachingAgent;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.caching.agent.KubernetesV2CachingAgent;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.KubernetesSpinnakerKindMap;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesKind;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest;
+import com.netflix.spinnaker.clouddriver.model.Manifest;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KubernetesMutatingWebhookConfigurationHandler extends KubernetesHandler {
+  @Override
+  public int deployPriority() {
+    return DeployPriority.ADMISSION_PRIORITY.getValue();
+  }
+
+  @Override
+  public KubernetesKind kind() {
+    return KubernetesKind.MUTATING_WEBHOOK_CONFIGURATION;
+  }
+
+  @Override
+  public boolean versioned() {
+    return false;
+  }
+
+  @Override
+  public KubernetesSpinnakerKindMap.SpinnakerKind spinnakerKind() {
+    return KubernetesSpinnakerKindMap.SpinnakerKind.UNCLASSIFIED;
```

Comment:
Didn't we declare a `Kind` for mutating webhooks?

Reply:
This is the spinnaker kind it maps to, not the Kubernetes kind. On startup, this spinnaker -> kubernetes kind relation is recorded, and informs both the resource providers (e.g. cluster provider, app provider, instance provider) how to serve these resources, as well as the UI on how to render them.